CriteoPublisherSdk/Sources/DataConsent/CR_Gdpr.h

```diff
@@ -65,6 +77,14 @@ typedef NS_ENUM(NSInteger, CR_GdprTcfVersion) {
  */
 - (BOOL)isConsentGivenForPurpose:(NSUInteger)id;
 
+/**
+ * Gives the Criteo publisher restrictions for a given purpose
+ *
+ * @param id Purpose id
+ * @return Publisher Restriction type
+ */
+- (CR_GdprTcfPublisherRestrictionType)publisherRestrictionsForPurpose:(NSUInteger)id;
```

Comment:
AFAIU, we're only using this method with id=1. Is there any reason why this is parameterized and not hardcoded ?

Reply:
Nvm, I saw that you were using purpose=7 before.